tests/playbooks/tests_802_1x.yml

```diff
@@ -122,3 +127,8 @@
               command: update-ca-trust
           tags:
             - "tests::cleanup"
+
+- name: Restore host state
```

Comment:
For other cleanup tasks, we added them in an always block to ensure that they always run. Do we need this here, too?